pkg/placement/membership.go

```diff
@@ -307,9 +307,11 @@ func (p *Service) performTableDissemination() {
 func (p *Service) performTablesUpdate(hosts []placementGRPCStream, newTable *v1pb.PlacementTables) {
 	// TODO: error from disseminationOperation needs to be handle properly.
 	// Otherwise, each Dapr runtime will have inconsistent hashing table.
-	p.disseminateOperation(hosts, "lock", nil)
-	p.disseminateOperation(hosts, "update", newTable)
```

Comment:
by removing this you're allowing the sidecars to update with the latest table entries.

Reply:
My idea is to change from locking all hosts first to locking each host in turn for efficiency.

Reply:
It is more like a CP model as currently implemented, and more like an AP model after the change. Because we don't need every DAPR client actor to have the same data at the same time, as long as he has data that the client can call